registries/docker-daemon/start.sh

```diff
@@ -2,12 +2,10 @@
 
 # Enable local registry
 echo "Installing a daemon registry"
-docker run -d -p 80:5000 registry:2
+docker run -d -p 5000:5000 --name registry registry:2
```

Comment:
why switch the port? This is how we document using the local registry.

Reply:
since we're now using the IP of the registry from inside Docker, it's easier to keep the port number consistent.

Reply:
Consumers of FATS shouldn't know or care where the registry is located.